### DISCUSSION TITLE
Beginner looking for a place to help out!

### DISCUSSION THREAD

**Discussion by lillastormen** [jeremyckahn/farmhand#520] [General]
Hi there,

I'm at the start of my GitHub journey and this seems like a good place to start! :) I'd like to contribute in any way but not really sure where to begin. Any chance you can point me in the right direction? 

Cheers!

**Comment by jeremyckahn** [comment]
Hi @lillastormen, thanks for offering to help with Farmhand development! Contributions are welcome from anyone and you’re free to work on any part of the game that you’d like. I’d recommend starting with a small feature addition. You can either propose your own feature idea or pick up an issue from the backlog that seems interesting. In particular, I recommend looking at [issues with the “good first issue” label](https://github.com/jeremyckahn/farmhand/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).

Once you’re more familiar with the Farmhand code base and technology it uses (HTML/CSS/JavaScript/[React](https://react.dev/)/[MUI](https://mui.com/material-ui/getting-started/)), you could start looking at the [other issues in the backlog](https://github.com/jeremyckahn/farmhand/issues) to work on. Again, you're always free to propose your own idea to add to the game.

I'd also recommend joining [the Farmhand Discord server](https://discord.gg/6cHEZ9H). That's where the devs and regular players hang out and it would be great to have you join the community.

Let me know if you have any additional questions!
